pkg/cmdlets/errors/errors.go

```diff
@@ -93,8 +93,6 @@ func ParseUserFacing(err error) (int, error) {
 		return 0, nil
 	}
 
-	_, hasMarshaller := err.(output.Marshaller)
```

Comment:
This change is breaking the original intend of the function. Just way predated the new design and isn't looking to address the same use-case. What you effectively found here is that the term "User Facing" was already in use, and we need to either resolve the conflict of overlapping terms, or ensure that they are compatible.

Looking through the code it seems `hasMarshaller` will mainly identify whether we already have an OutputError on our hands, so we don't redundantly wrap it. While I don't see any scenario's in which this could be true, do we want to risk breaking this behaviour for this PR?

Please take some time to understand what OutputError{} is doing and how the new concept of user facing errors plays into this.

Reply:
From what I can grok the `OutputError` type adds extra post-processing and the handling in this function ensures that errors returned all have a similar UX. I've wrapped the user-facing error in this `OutputError` type as well to ensure it's consistent with all of the other errors we are presenting to users.

I've also added back the `hasMarshaller` check as you were right, we don't want to wrap the error again in the `OutputError` type.